src/mmrelay/migrate.py

```
@@ -0,0 +1,1832 @@
+                if dest.exists():
+                    shutil.rmtree(str(dest))
+                    logger.debug(
+                        "Removing existing %s plugin for move: %s", tier_name, dest
+                    )
+                shutil.move(str(item), str(dest))
+                logger.debug("Migrated %s plugin: %s", tier_name, item)
+                migrated = True
+            except (OSError, IOError) as e:
+                logger.warning("Failed to migrate %s plugin %s: %s", tier_name, item, e)
+                errors.append(f"{tier_name} {item}: {e}")
+        return migrated
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)